src/layers/core/point-cloud-layer/point-cloud-layer-vertex-64.glsl.js

```diff
@@ -31,7 +31,7 @@ attribute vec3 instancePickingColors;
 
 uniform float renderPickingBuffer;
 uniform float opacity;
-uniform float radius;
+uniform float radiusPixels;
```

Comment:
point-cloud-layer-vertex-64.glsl was never hit as we failed the projectionMode check.